src/genetic_forensic_portal/app/client/gf_api_client.py

```diff
@@ -98,3 +106,32 @@ def list_completed_analyses() -> list[str]:
     # and we can return its response
 
     return UUID_LIST
+
+
+def get_analysis_status(sample_id: str | None) -> str:
```

Comment:
Doesn't make a ton of sense to accept a `None` value for `sample_id` here, so it's fine to just remove it

```suggestion
def get_analysis_status(sample_id: str) -> str:
```